### ISSUE TITLE
no function with name 'COLUMN_OT' exists in this scope

### ISSUE THREAD

**Issue by rimblas** [plug-ins-pro/enhanced_grid_pro#1]
We just installed the grid and the UC_GRID package is invalid, and we're getting the following error:

```
Errors for PACKAGE BODY UC_GRID:

LINE/COL ERROR
-------- -----------------------------------------------------------------
1121/13  PL/SQL: Statement ignored
1121/28  PLS-00222: no function with name 'COLUMN_OT' exists in this scope
2393/21  PL/SQL: Statement ignored
2393/36  PLS-00222: no function with name 'COLUMN_OT' exists in this scope
```

**Comment by DougAGault** [comment]
Looking at the code, the assignments that are failing look like this 

`L_COLUMN_OT := COLUMN_OT();`

For some reason this is failing. However, you can use a more rudimentary 'Blunt force' way to do this by using 

`L_COLUMN_OT := NULL;`

This will still blank out the record but doesn't cause the error.

**Comment by agrlica** [comment]
Hi guys,

Can you guys please provide information on which Database and APEX version you are testing it? 
We try to gather as much information as possible, thank you!

I can confirm that on 
Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production 
was working without any problem.


Nevertheless, I've updated the package. The new patch version should be out soon.
To give more specifics on a new release, improvements with LOVs also some due to APEX 23.1 will be also released soon (few weeks).


Kind regards,
Andrej Grlica

**Comment by DougAGault** [comment]
Hey Andrej,

Looking at the code, I was pretty surprised that it didn't work. I even went back to the DB documentation and saw references to that syntax in V11 of the database 

Here's our versions. 
* APEX 22.2.1
* DB  12.2.0.1.0 Enterprise Edition

Just so you know, we install all our plugin code in a central schema and then grant access out to the schema's that need it. Not sure if that would make a difference at all. 

Let us know if you need any more information.

**Comment by agrlica** [comment]
Thank you for the update.
For quick fix
```
L_COLUMN_OT := NULL;
```
will be a workaround.
We will put more time to test this when Andraž comes back from vacation.

**Comment by DougAGault** [comment]
Fixed with latest beta release of 23.2

**Comment by rimblas** [comment]
!thank you
